src/App.css

```diff
@@ -53,5 +105,112 @@ tr {
 }
 
 .card {
-  width: 18rem;
+  background-color: #f0f0f0;
+  width: 22rem;
+  border: none;
+  height: fit-content;
+  padding-bottom: 20px;
+  text-align: center;
+  transition: 200ms;
+}
+.card:hover {
+  transform: scale(1.01);
+}
+.Image_CSS {
```

Comment:
I'm not sure the `_CSS` bit is necessary? When I'm writing code, I usually only write classes to be used in CSS, so writing it for every class is not needed. What do you think?

Reply:
some time our project is big and we have a lots of code I separated id and class with "_CSS" and "_JSX" only for that time that I'm looking to find something this would be helpful for me to find fast and better organizing my code, and this was one of the mentors advice